nibabel/externals/netcdf.py

```diff
@@ -607,7 +607,10 @@ def _read_var_array(self):
                     pos = self.fp.tell()
                     self.fp.seek(begin_)
                     data = fromstring(self.fp.read(a_size), dtype=dtype_)
-                    data.shape = shape
+                    # Avoid exception for empty variables.
```

Comment:
Should this fix go into the scipy version of the file, that we copied from?

Is there a test for this change?